spec/menu_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "menu"
+
+describe Menu do
```

Comment:
Good set up of doubles using dependency injection, and tests have been clearly ordered